### PR TITLE
Fix last mobile merge into master

### PR DESCRIPTION
**Pull request by Tug**
## Description
Last PR https://github.com/WordPress/gutenberg/pull/14503 was supposed to merge all the changes from `rnmobile/develop` (the development branch for mobile) to master. It seems I made some mistakes when resolving conflicts as a couple of changes from `rnmobile/develop` were omitted. This PR aims to fix this and have `rnmobile/develop` fully merged into `master`.

## How has this been tested?
I checked manually by looking at the PRs merged into `rnmobile/develop` and what we have in `master`.
I also used the following command to find diff between `master` and `rnmobile/develop`:
```
git diff -M origin/master...origin/rnmobile/develop --name-only --diff-filter=M | grep .native. | xargs git diff -M origin/master...origin/rnmobile/develop --
```
Note that I'm excluding all non `.native.*` file here.

### CODE REVIEW

File: packages/block-library/src/heading/style.native.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@import "variables.scss";
```
Comment (Tug) [inline packages/block-library/src/heading/style.native.scss]: We'll have to remove this, this was dropped as part of the merge PR (we auto import `_variables.scss` now)

File: packages/block-library/src/paragraph/style.native.scss [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@import "variables.scss";
```
Comment (Tug) [inline packages/block-library/src/paragraph/style.native.scss]: Same here

**Review by SergioEstevao — APPROVED** [review]
Looking good.

**Review by SergioEstevao — APPROVED** [review]
Looking good!

**Pull request merged**